specification/iotcentral/data-plane/readme.md

```diff
@@ -75,6 +75,14 @@ These settings apply only when `--tag=package-2022-07-31` is specified on the co
 input-file:
   - Microsoft.IoTCentral/stable/2022-07-31/iotcentral.json
 ```
+
+### Tag: package-2022-10-31-preview
+These settings apply only when `--tag=package-2022-10-31-preview` is specified on the command line.
```

Comment:
Normally, you'd want to update your top-level `tag:` to point to the newest release, preview or stable.